lib/runners/processor/pylint.rb

```diff
@@ -0,0 +1,98 @@
+module Runners
+  class Processor::Pylint < Processor
+    include Python
+
+    Schema = StrongJSON.new do
+      let :runner_config, Schema::BaseConfig.base.update_fields { |fields|
+        fields.merge!({
+          target: enum?(string, array(string)),
+          rcfile: string?,
+          ignore: enum?(string, array(string)),
+          'errors-only': boolean?,
+        })
+      }
+
+      let :rule, object(
+        severity: string,
+        'message-id': string,
+        module: string,
+        obj: string,
+      )
+    end
+
+    register_config_schema(name: :pylint, schema: Schema.runner_config)
+
+    DEFAULT_TARGET = ["**/*.{py}"].freeze
+
+    def analyze(changes)
+      run_analyzer
+    end
+
+    private
+
+    def rcfile
+      config_linter[:rcfile].then { |file| file ? ["--rcfile=#{file}"] : [] }
+    end
+
+    def ignore
+      Array(config_linter[:ignore] || []).then { |arr| arr.empty? ? nil : "--ignore=#{arr.join(",")}" }
+    end
+
+    def erros_only
+      config_linter[:'errors-only'].then { |value| value ? ["--errors-only"] : [] }
+    end
+
+    def analyzed_files
+      # Via glob
+      targets = Array(config_linter[:target] || DEFAULT_TARGET)
+      globs = targets.select { |glob| glob.is_a? String }
+      Dir.glob(globs, File::FNM_EXTGLOB, base: current_dir)
+    end
+
+    def parse_result(output)
+      json = JSON.parse(output, symbolize_names: true)
+      json.flat_map do |issue|
+        yield Issue.new(
+          id: issue[:symbol],
+          path: relative_path(issue[:path]),
+          location: Location.new(start_line: issue[:line]),
+          message: issue[:message],
+          object: {
+            severity: issue[:type],
+            'message-id': issue[:'message-id'],
+            module: issue[:module],
+            obj: issue[:obj],
+          },
+          schema: Schema.rule,
+        )
+      end
+    end
+
+    def run_analyzer
+      files = analyzed_files
+
+      trace_writer.message "Analyzing #{files.size} file(s)..."
+
+      stdout, stderr = capture3!(
+        analyzer_bin,
+        *files,
+        *rcfile,
+        *ignore,
+        *erros_only,
+        '--output-format=json',
+        '--jobs=2',
+        '--exit-zero',
+        '--disable=E0401',
```

Comment:
[question] Always disable `E0401`? Why?

Reply:
>Maybe, we should add --disable=import-error in the command line arguments because the violation import-error would be issued against almost all projects. 

`E0401` is the message-id of `import-error` symbol.

Reply:
I see. Then, can you please add a comment and use `import-error` instead of `E0401`?
It's better to add a link to a document about Pylint's importing modules.

```suggestion
        # NOTE: We don't support importing modules yet. See https://...
        '--disable=import-error',
```